src/SolutionTemplate/UnoSolutionTemplate.Wizard.2022/UnoSolutionWizard.cs

```diff
@@ -373,33 +373,45 @@ private void SetStartupProject()
 			}
 		}
 
-		private void SetUWPAnyCPUBuildableAndDeployable()
+		private void SetBuildableAndDeployable()
 		{
 			ThreadHelper.ThrowIfNotOnUIThread();
 
-			if (_dte?.Solution.SolutionBuild is SolutionBuild2 val)
+			if (_dte?.Solution.SolutionBuild is SolutionBuild2 solutionBuild)
 			{
-				try
-				{
-					var anyCpuConfig = val.SolutionConfigurations
-						.Cast<SolutionConfiguration2>()
-						.FirstOrDefault(c => c.Name == "Debug" && c.PlatformName == "Any CPU");
+				SetConfiurationDeployable(solutionBuild, "Debug", "Any CPU", ".Windows.csproj");
+				SetConfiurationDeployable(solutionBuild, "Release", "Any CPU", ".Windows.csproj");
+
+				SetConfiurationDeployable(solutionBuild, "Debug", "Any CPU", ".Mobile.csproj");
```

Comment:
Please also make iOS project deployable for iPhone configuration - should be quick add here - https://github.com/unoplatform/uno/issues/9572

Reply:
There's no iPhone configuration on net6, but although there is one in Xamarin projects, there's no "deploy" option available there.

The current change make all targets buildable and deployable, so it should fix #9572.